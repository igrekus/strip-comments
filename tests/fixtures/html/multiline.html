<p>text</p>
<!--
first line
second line
third line
-->
<p>More text</p>
<!--
first line
second line
third line
-->
