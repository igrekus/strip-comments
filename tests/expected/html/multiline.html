<p>text</p>

<p>More text</p>

